platformio.ini

```diff
@@ -487,7 +487,7 @@ build_flags = ${common.build_flags}
 lib_deps =
   U8glib-HAL=https://github.com/MarlinFirmware/U8glib-HAL/archive/bugfix.zip
   LiquidCrystal@1.3.4
-  TMCStepper@>=0.5.0,<1.0.0
+  https://github.com/bigtreetech/TMCStepper#master
```

Comment:
prefer this format : the team prefer to avoid git requirement....
TMCStepper=https://github.com/bigtreetech/TMCStepper/archive/master.zip